### ISSUE TITLE
Patch operations do not work with no initial declaration

### ISSUE THREAD

**Issue by mstokes-f5** [F5Networks/f5-appsvcs-extension#508]
### Environment
 * Application Services Version: 3.30.0
 * BIG-IP Version: 15.1.2.1

### Summary
AS3 patch operations (either a `PATCH` request to `/declare` or a `patch` `action` in a `POST`) fail with a strange error (invalid declaration: '/body should be object') when no initial declaration is present (e.g., on a fresh install or after a `DELETE` to `/declare`).

### Steps To Reproduce
1. Start with a fresh install or send a `DELETE` request to `/declare` endpoint
2. `POST` this declaration:
```json
{
    "class": "AS3",
    "id": "fast%create%foo%bar%0",
    "action": "patch",
    "patchBody": [
        {
            "op": "add",
            "path": "/id",
            "value": "fast%create%foo%bar%0"
        },
        {
            "op": "add",
            "path": "/foo",
            "value": {
                "class": "Tenant",
                "bar": {
                    "class": "Application",
                    "template": "udp",
                    "serviceMain": {
                        "class": "Service_UDP",
                        "virtualAddresses": [
                            "192.0.2.1"
                        ],
                        "virtualPort": 5555,
                        "pool": "bar_Pool1"
                    },
                    "bar_Pool1": {
                        "class": "Pool",
                        "monitors": [
                            "icmp"
                        ],
                        "members": [
                            {
                                "serverAddresses": [
                                    "192.0.2.2"
                                ],
                                "servicePort": 5555
                            }
                        ]
                    }
                }
            }
        }
    ]
}
```

### Expected Behavior
Either:
1. The patch succeeds with a new tenant and application added to an otherwise empty declaration (preferred)
2. An error response saying patch operations require an initial POST with a declaration

### Actual Behavior
A `422` error response with `declaration is invalid` (`"/body: should be object"`).

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added it to our internal product backlog as AUTOTOOL-2738. I think we should be able to provide behavior 1 from the expected behavior. If we cannot, then we will fallback to option 2 of improving the error message.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.32.0 release.
